matplotlib-chord.py

```diff
@@ -162,13 +164,14 @@ def chordDiagram(X, ax, colors=None, width=0.1, pad=2, chordwidth=0.7):
     x = X.sum(axis = 1) # sum over rows
     ax.set_xlim(-1.1, 1.1)
     ax.set_ylim(-1.1, 1.1)
-
-    if colors is None:
-    # use d3.js category10 https://github.com/d3/d3-3.x-api-reference/blob/master/Ordinal-Scales.md#category10
+    
+    import matplotlib.pyplot as plt
```

Comment:
`plt` has already been imported in line 3.

Reply:
Sure! I'll change that.